ouroboros-consensus-cardano/src/ouroboros-consensus-cardano/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -730,7 +730,11 @@ protocolInfoCardano paramsCardano
         praosMaxMajorPV = maxMajorProtVer,
         praosMaxLovelaceSupply = SL.sgMaxLovelaceSupply genesisShelley,
         praosNetworkId = SL.sgNetworkId genesisShelley,
-        praosSystemStart = SystemStart $ SL.sgSystemStart genesisShelley
+        praosSystemStart = SystemStart $ SL.sgSystemStart genesisShelley,
+        praosRandomnessStabilisationWindow =
+          SL.computeRandomnessStabilisationWindow
```

Comment:
Maybe add a comment here, at least saying `see 'partialConsensusConfigBabbage'` or something like that